cni/network/invoker_cns.go

```diff
@@ -21,11 +21,17 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	ExpectedNumInterfacesWithDefaultRoutes = 1
```

Comment:
why this is exported variable. can it be  `expectedNumInterfacesWithDefaultRoutes`?